### ISSUE TITLE
Cannot save a description edit

### ISSUE THREAD

**Issue by reubenosborne1** [archesproject/arches#4317]
**Describe the bug**
 There is not option to save an edit to a description once you've clicked another card. See attached gif. You can make and save as many edits you like if you do not click off the description card.

![descriptionerror](https://user-images.githubusercontent.com/41572010/48940764-b6314e80-ef10-11e8-8b6e-b95b5db13134.gif)

**To Reproduce**
1. Make a new resource with a description. 
2. Add some more data in other cards
3. Go back to description to make an edit. 
4. Cannot save! 

**Your Arches Information**

*   Version used: 4.3.3
*   Operating System and version (desktop or mobile): Ubuntu 16.04.5
*   Browser Name and version: Google Chrome
*   Link to your Arches Install (optional):

**Comment by azaroth42** [comment]
Screencast demonstrates that the bug is real. Potentially related to #4113 and the algorithm to determine the "dirty state" of the editor in the browser. Recommend attention, as likely to affect other instances.

**Comment by reubenosborne1** [comment]
This issue is still present in stable/4.4.2. 

1. I have made a description entry and clicked save... 
![description1](https://user-images.githubusercontent.com/41572010/65043115-2f8fa700-d952-11e9-99b5-9c67c98d54eb.png)


2. I have clicked onto a different card and back to the description entry to make an edit. I added the text *making an edit* and no save button appears... 
![edited_desc](https://user-images.githubusercontent.com/41572010/65043118-31596a80-d952-11e9-8ff7-6b1599355732.png)
